integrations/pom.xml

```diff
@@ -221,7 +221,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>job-dsl</artifactId>
-      <version>1.72</version>
+      <version>1.73.1-SNAPSHOT</version>
```

Comment:
normally a timestamped snapshot if no incrementals so you know what its running 😉

Reply:
I tried... job-dsl-core would not restore with timestamped

Reply:
I tried... job-dsl-core would not restore with timestamped (hint they don't have the same timestamp) @daspilker (@sghill current maintainer on gradle-jpi-plugin)

Reply:
Is the issue that there are multiple modules and they end up with slightly different timestamps in the manifest?

I like creating release candidates for this scenario. Gradle-jpi-plugin will [take the given version unless it ends in `-SNAPSHOT`](https://github.com/jenkinsci/gradle-jpi-plugin/blob/master/src/main/java/org/jenkinsci/gradle/plugins/jpi/internal/VersionCalculator.java#L31), so it's possible to create a `1.73.0-rc.1` if you'd like. The `nebula.release` plugin makes this easy with its [`candidate` task](https://github.com/nebula-plugins/nebula-release-plugin#tasks-provided).

Reply:
> Is the issue that there are multiple modules and they end up with slightly different timestamps in the manifest?

I believe it is. You can explore the pom here: https://repo.jenkins-ci.org/webapp/#/search/quick/eyJzZWFyY2giOiJxdWljayIsInF1ZXJ5Ijoiam9iLWRzbCotMS43My4xKi5wb20ifQ==
You can hover over the artifact and download/view it.